fern/docs/pages/cli/fern-init.mdx

```diff
@@ -40,4 +40,4 @@ colors:
 ```
 </CodeBlock>
 
-To see what the API docs looks like, run `fern generate --docs`. More details on this command are available [here](/fern-generate).
\ No newline at end of file
+To publish the API docs, run `fern generate --docs`. See more details on this command at [fern-generate](https://docs.buildwithfern.com/overview/cli/fern-generate).
```

Comment:
this is an awesome change that gives the reader more of an idea of where they're headed than writing "here"! kudos